package.json

```
@@ -12,6 +12,7 @@
     "@iconify-json/material-symbols": "1.2.18",
     "@iconify-json/simple-icons": "1.2.31",
     "@iconify/react": "5.2.1",
+    "@next/third-parties": "^15.3.2",
```

動作安定のため，依存関係のバージョンは固定でお願いします🙏

```suggestion
    "@next/third-parties": "15.3.2",
```